README.md

```diff
@@ -1,11 +1,31 @@
-# eSchool
-Microservice based eSchool Solution
+# eSchool - Microservice and Containers based open source School Administration Software
 
 ## Linux Build Status for 'master' branch
 
 |                                                                                                    Identity API                                                                                                    |
 | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
 | [![Build Status](https://dev.azure.com/OpenCodeFoundation/eSchool/_apis/build/status/identity?branchName=master)](https://dev.azure.com/OpenCodeFoundation/eSchool/_build/latest?definitionId=1&branchName=master) |
 
+## Tech & Tools
+- .NET Core
+- Docker
+- Orchestrators: Kubernetes
+- Visual Studio
+- MongoDB
+- SQL Server
+- Azure DevOps
+- Redis
+- RabbitMQ
 
+## Updated for .NET Core 2.2 "wave" of technologies
+eSchool is updated to .NET Core 2.x (currently updated to 2.2) "wave" of technologies. Not just compilation but also new recommended code in EF Core, ASP.NET Core, and other new related versions.
 
+> ### DISCLAIMER
+> **IMPORTANT:** The current state of this application is **BETA**, because we are constantly evolving towards newly released technologies. Therefore, many areas could be improved and change significantly while refactoring the current code and implementing new features. Feedback with improvements and pull requests from the community will be highly appreciated and accepted.
+>
+> This reference application proposes a simplified microservice oriented architecture implementation to introduce technologies like .NET Core with Docker containers through a comprehensive application. The chosen domain is eSchool/Online Scchool administration software but simply because it is a well-known domain by most people/developers.
```

Comment:
We are not trying to develop reference application here. We are trying to develop a software application that everyone can use in production by following the `eShopOnContainer` reference application